### PR TITLE
New EasyConfig for SQLite 3.10.0 with the foss-2015a toolchain

### PR DESCRIPTION
**Pull request by valtandor**
Required for the GRASS 7.0.2 dependency tree.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/12cffb4f955d1d91071e for a full test report.

File: easybuild/easyconfigs/s/SQLite/SQLite-3.10.0-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+version_str = '%(version_major)s' + ''.join('%02d' % int(x) for x in version_minor_etc)
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/sqlite3'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.10.0-foss-2015a.eb]: @valtandor: please merge https://github.com/valtandor/easybuild-easyconfigs/pull/6 to extend sanity check (like in existing SQLite easyconfigs)

**Comment by valtandor** [comment]
Merge done.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f5615e45b2b8d14a6a0d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**